src/renderer/components/Home.js

```diff
@@ -256,50 +288,85 @@ export default function Home ({ onSelectLanguage }) {
 
   return (
     <Root>
-      <Sidebar>
-        <TitleBarShim />
-        <Logo>
-          <MapeoIcon fontSize='large' />
-          <div>
-            <h1>Mapeo</h1>
-            {buildConfig.variant === 'icca' ? <h2>for ICCAs</h2> : null}
-          </div>
-        </Logo>
-        <StyledTabs
-          orientation='vertical'
-          variant='scrollable'
-          value={tabIndex}
-          onChange={(e, value) => setTabIndex(value)}
-        >
-          <StyledTab icon={<MapIcon />} label={t(m.mapeditor)} />
-          <StyledTab icon={<ObservationIcon />} label={t(m.mapfilter)} />
-          <StyledTab icon={<SyncIcon />} label={t(m.sync)} />
-          {hasUpdate && (
+      <div style={{ display: 'flex', flex: 1 }}>
+        <Sidebar>
+          <TitleBarShim />
+          <Logo>
+            <MapeoIcon fontSize='large' />
+            <div>
+              <h1>Mapeo</h1>
+              {buildConfig.variant === 'icca' ? <h2>for ICCAs</h2> : null}
+            </div>
+          </Logo>
+          <StyledTabs
+            style={{ height: '100%' }}
+            className={classes.root}
+            orientation='vertical'
+            variant='scrollable'
+            value={tabIndex}
+            onChange={(e, value) => {
+              if (value === 4) setSettingsReset(true)
+              setTabIndex(value)
+            }}
```

Comment:
a little bit hard to visualize what's going on here. when does `value === 4`? is there a more descriptive/explicit way to fulfill the same condition?

Reply:
Sort of. I can explicitly name the tab. And the i can say if ``value` === that name. But there is no type safety to it, and all the other tabs are using the index. I want to change this component to have type safety....but styled components is [not letting that happen](https://stackoverflow.com/questions/71855570/styled-components-could-not-be-used-as-jsx-element-react-typescript):

Ill change it to be more explicit, but this whole page (or all of desktop realistically )....

Reply:
ah i see, `4` is passed as an explicit value to one of the tabs. in that case, using a constant defined outside of the component would make this easier to understand at the very least. I thought the value was referring to the index position of the tab, which would've made it trickier and less reliable